relayer/message_coordinator.go

```diff
@@ -76,6 +77,7 @@ func (mc *MessageCoordinator) getAppRelayerMessageHandler(
 	}
 
 	// Fetch the message delivery data
+	//nolint:lll
 	sourceBlockchainID, originSenderAddress, destinationBlockchainID, destinationAddress, err := messageHandler.GetMessageRoutingInfo()
```

Comment:
Not for this PR, but these are the exact fields that compose a `RelayerID`. It might make sense to directly return that struct instead of the individual fields.